crates/holochain/tests/tests/multi_conductor/mod.rs

```diff
@@ -45,75 +45,83 @@ async fn dpki_no_publish() {
         .unwrap();
 }
 
-/* @ K2-INTEGRATION @ TODO @
 /// Test that op publishing is sufficient for bobbo to get alice's op
 /// even with gossip disabled.
 #[cfg(feature = "test_utils")]
 #[tokio::test(flavor = "multi_thread")]
-async fn test_publish() -> anyhow::Result<()> {
-    use std::sync::Arc;
-
-    use holochain::test_utils::inline_zomes::simple_create_read_zome;
-    use kitsune_p2p_types::config::KitsuneP2pConfig;
+async fn test_publish() {
+    use holochain::{retry_until_timeout, test_utils::inline_zomes::simple_create_read_zome};
+    use holochain_conductor_api::conductor::{ConductorConfig, NetworkConfig};
+    use kitsune2_api::DhtArc;
 
     holochain_trace::test_run();
-    const NUM_CONDUCTORS: usize = 3;
-
-    let (signal_url, _signal_srv_handle) = kitsune_p2p::test_util::start_signal_srv().await;
-
-    let mut tuning =
-        kitsune_p2p_types::config::tuning_params_struct::KitsuneP2pTuningParams::default();
-    tuning.gossip_strategy = "none".to_string();
 
-    let mut network = KitsuneP2pConfig::from_signal_addr(signal_url);
-    network.tuning_params = Arc::new(tuning);
     let config = ConductorConfig {
-        network,
-        tuning_params: Some(ConductorTuningParams {
-            sys_validation_retry_delay: Some(std::time::Duration::from_millis(100)),
-            countersigning_resolution_retry_delay: None,
+        network: NetworkConfig {
+            disable_gossip: true,
             ..Default::default()
-        }),
-        dpki: DpkiConfig::disabled(),
+        },
         ..Default::default()
     };
 
-    let mut conductors = SweetConductorBatch::from_config(NUM_CONDUCTORS, config).await;
-
-    let (dna_file, _, _) =
-        SweetDnaFile::unique_from_inline_zomes(("simple", simple_create_read_zome())).await;
-
+    let mut conductors = SweetConductorBatch::from_config_rendezvous(2, config).await;
+    let dna_file = SweetDnaFile::unique_from_inline_zomes(("simple", simple_create_read_zome()))
+        .await
+        .0;
     let apps = conductors.setup_app("app", &[dna_file]).await.unwrap();
-    conductors.exchange_peer_info().await;
+    let ((alice,), (bobbo,)) = apps.into_tuples();
 
-    let ((alice,), (bobbo,), (carol,)) = apps.into_tuples();
+    // Set full storage arc for both peers and then exchange peer infos.
+    conductors[0]
```

Comment:
Can this now use the helper in Sweettest?

Reply:
It can. I didn't use it here because it'd be slightly inefficient to first wait for one and then for the other... but that doesn't matter here, does it?!